src/shared/components/case-viewer/case-review-specific-access-request/case-review-specific-access-request.component.html

```diff
@@ -0,0 +1,128 @@
+<exui-error-message
+  *ngIf="formGroup.invalid && submitted"
+  [error]="errorMessage">
+</exui-error-message>
+
+<form [formGroup]="formGroup" (submit)="onSubmit()">
+  <div class="govuk-form-group">
+    <fieldset class="govuk-fieldset" aria-describedby="reason-hint">
+      <div class="govuk-grid-column-two-thirds" >
+        <p></p>
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+          <h1 class="govuk-fieldset__heading">
+            {{ title }}
+          </h1>
+        </legend>
+        <p></p>
+        <table class="govuk-table">
+          <caption></caption>
+          <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header leftLabel" >
+                Case name
+              </th>
+              <td class="govuk-table__cell">{{ requestAccessDetails.caseName }}</td>
+            </tr>
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header leftLabel" >
+                Case reference
+              </th>
+              <td class="govuk-table__cell">{{ requestAccessDetails.caseReference }}</td>
+            </tr>
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header leftLabel" >
+                Date submitted
+              </th>
+              <td class="govuk-table__cell">{{ requestAccessDetails.dateSubmitted }}</td>
+            </tr>
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header leftLabel" >
+                Request from
+              </th>
+              <td class="govuk-table__cell">{{ requestAccessDetails.requestFrom }}<br /></td>
+            </tr>
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header leftLabel" >
+                Reason for case access
+              </th>
+              <td class="govuk-table__cell">{{ requestAccessDetails.reasonForCaseAccess }}</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <div
+          class="govuk-form-group"
+          [ngClass]="{ 'form-group-error': formGroup.invalid && submitted }">
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+              <h1 class="govuk-fieldset__heading">
+                {{ hint }}
+              </h1>
+            </legend>
+            <div
+              id="error-message"
+              class="govuk-error-message"
+              *ngIf="formGroup.get('radioSelected').invalid && submitted">
+              {{ errorMessage.description }}
+            </div>
+            <div class="govuk-radios__item">
```

Comment:
The only difference with the three reasons is the index, so can this block be rewritten using a loop?

Reply:
possible , but to keep same format with other radios that is implimented same way.